epictrack-api/src/api/services/event.py

```diff
@@ -45,15 +46,14 @@
 from api.utils import util
 from api.application_constants import MIN_WORK_START_DATE
 
+from api.config import _Config
```

Comment:
We can get the current config for the environment by
```
from flask import current_app
config = current_app.config
```
This way can avoid importing a 'private' member.